pkg/policy/middleware.go

```diff
@@ -75,6 +75,21 @@ func (m *Middleware) EnforceAccess(p *Policy, handler http.HandlerFunc) http.Han
 	}
 }
 
+func (m *Middleware) EnforceLicense(handler http.HandlerFunc) http.HandlerFunc {
```

Comment:
Not really.  That won't bail with 403.   This follows the pattern with `RequireValidSessionMiddleware`/`EnforceAccess` where first one loads info and second one enforces it... Don't really know why it's structured that way.   I just didn't want to mix this logic with what's there since it's not really related.